pyproject.toml

```diff
@@ -1,3 +1,3 @@
 [build-system]
-requires = ["setuptools>=61.0","pip"]
+requires = ["setuptools>=61.0"]
```

Comment:
To clarify a bit, the `build-system.requires` are tools that get installed from PyPI into a temporary virtual environment that the build happens in which is then cleaned up and removed. So `pip` isn't something that should get added here (unless you had to intentionally downgrade `pip`, but at that point you have what amounts to a broken build and you should fix your build instead).

c.f. https://learn.scientific-python.org/development/guides/packaging-classic/#pep-517518-support-high-priority for more high level details or read PEP 517/518.